satpy/readers/utils.py

```diff
@@ -474,3 +475,53 @@ def remove_earthsun_distance_correction(reflectance, utc_date=None):
     with xr.set_options(keep_attrs=True):
         reflectance = reflectance / reflectance.dtype.type(sun_earth_dist * sun_earth_dist)
     return reflectance
+
+
+def get_distributed_friendly_dask_array(manager, varname, chunks, dtype,
+                                        group="/", auto_maskandscale=None):
```

Comment:
I'm not sure how I feel about this function name. Obviously it makes sense in this PR because it solves this specific problem, but it feels like there is a (shorter) more generic name that gets the point across. Another thing is that `distributed_friendly` is mentioned here, but that friendliness is a side effect of the "serializable" nature of the way you're accessing the data here, right? `get_serializable_dask_array`?

I don't feel super strongly about this, but the name was distracting to me so I thought I'd say something.

Reply:
Renamed `get_serializable_dask_array`.